Platform/CoffeelakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -839,6 +839,71 @@ STATIC S3_SAVE_REG mS3SaveReg = {
   { { IO, WIDE32, { 0, 0}, (ACPI_BASE_ADDRESS + R_ACPI_IO_SMI_EN), 0x00000000 } }
 };
 
+VOID
+ProcessAllLocks (
+  IN VOID
+)
+{
+  SILICON_CFG_DATA  *SiliconCfgData;
+  UINT32            LpcBase;
+  UINTN             SpiBaseAddress;
+  UINT16            TcoBase;
+  UINTN             SpiBar0;
+
+  SiliconCfgData = (SILICON_CFG_DATA *)FindConfigDataByTag (CDATA_SILICON_TAG);
+  if ( (SiliconCfgData != NULL) && (SiliconCfgData->ECEnable == 1)){
+    //
+    // Enable decoding of I/O locations 62h and 66h to LPC
+    //
+    LpcBase = MmPciBase (0, PCI_DEVICE_NUMBER_PCH_LPC, 0);
```

Comment:
these are not locks.   Maybe we should re-arrange the code.

Reply:
We can just re-name the function probably? Or I can move the code back to ReadyToBoot phase instead of having a function.

Reply:
Either way works for me.